### DISCUSSION TITLE
Why does installing this package downgrade psr/log, filp/whoops and PHPmailer?

### DISCUSSION THREAD

**Discussion by benshawuk** [spatie/icalendar-generator#91] [Bugs]
I wanted to try this package, but after installing I get a load of whoops errors.
When uninstalling, I can see that it's re-upgrading back to newer versions:

  - Removing spatie/icalendar-generator (2.3.3)
  - Upgrading psr/log (1.1.4 => 3.0.0): Extracting archive
  - Upgrading filp/whoops (2.14.1 => 2.14.5): Extracting archive
  - Upgrading phpmailer/phpmailer (v6.5.1 => v6.6.3): Extracting archive

Does anyone know how to get around this issue?
Many thanks.

**Comment by rubenvanassche** [comment]
Did you install the package with or without dev dependencies? Could be that one of our dev dependencies causes this behavior.
